TravelPro.html

```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TravelPro</title>
+    <link rel="stylesheet" href="TravelPro.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+    <div class="navbar">
+        <div id="logo">
+            <img src="./images/logo.png" alt="logo" height="80px" width="110px">
+        </div>
+        <div id="options">
+            <a>Destinations</a>
+            <a>Hotels</a>
+            <a>Flights</a>
+            <a>Bookings</a>
+            <a>Login</a>
+            <a id="sign-up">Sign Up</a>
+            <select id="languages">
+                <option value="EN" selected><small>EN</small></option>
+                <option value="HIN"><small>HIN</small></option>
```

Comment:
The use of `<small>` tags within `<option>` elements is unusual. Styling the text size should be handled via CSS.